addons/mrp/wizard/change_production_qty.py

```diff
@@ -98,14 +98,14 @@ def change_prod_qty(self):
                 if production.product_id.tracking == 'serial':
                     quantity = 1.0 if not float_is_zero(quantity, precision_digits=precision) else 0.0
                 else:
-                    quantity = quantity if (quantity > 0) else 0
+                    quantity = quantity if float_compare(quantity, 0, precision_digits=precision) > 0 else 0
```

Comment:
We should use the mo_id.product_uom_id.rounding in that case no?

Reply:
Same here, I'm not sure. Considering the definition of `wo.qty_produced` (which is used to compute `quantity`):
https://github.com/odoo/odoo/blob/81497125d8100c6bdd2dc30434232a88a419a3e3/addons/mrp/models/mrp_workorder.py#L49-L53